examples/workflow_task_rolling.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
If we don't want to plot something, we don't have to use  jupyter notebook. 
Python scripts is easier to maintain.